sklearn/tree/tree.py

```diff
@@ -239,7 +239,7 @@ def fit(self, X, y, sample_weight=None, check_input=True,
         if max_depth <= 0:
             raise ValueError("max_depth must be greater than zero. ")
         if not (0 < max_features <= self.n_features_):
-            raise ValueError("max_features must be in (0, n_features]")
+            raise ValueError("max_features must be in (0, n_features)")
```

Comment:
Thanks a lot for opening a PR! I believe this is a mathematical notation to indicate where the bounds of the range are included or not. In this case, `(0, n_features]` means 0 is not included and n_features is included.

I am going to close this PR but feel free to find an issue to work on at: https://github.com/scikit-learn/scikit-learn/issues?labels=Easy. Have a look at our [contribution guidelines](https://github.com/scikit-learn/scikit-learn/blob/master/CONTRIBUTING.md) for more details.